src/Abstractions/Converters/JsonDataConverter.cs

```diff
@@ -8,24 +8,19 @@ namespace Microsoft.DurableTask.Converters;
 /// <summary>
 /// An implementation of <see cref="DataConverter"/> that uses System.Text.Json APIs for data serialization.
 /// </summary>
-public class JsonDataConverter : DataConverter
+/// <remarks>
+/// Initializes a new instance of the <see cref="JsonDataConverter"/> class.
+/// </remarks>
```

Comment:
Is `<remarks>` the recommended way to describe the constructor now? How are the class remarks disambiguated from constructor remarks in the generated .NET documentation?

Reply:
Removed all those `<remarks>`